### ISSUE TITLE
[Suggestion] Reimplement Archmage Armour

### ISSUE THREAD

**Issue by TheIcyOne** [Growlith1223/ArsMagica2#242]
In Ars Magica 2, Archmage armour was a diamond-tier armour set providing defensive buffs on specific contingencies, like water breathing while submerged and slowfall while falling.

**Comment by ProfHugo** [comment]
yes battlemage armor is pretty broken rn, it's nothing more than a set of reduced diamond armor
